### PR TITLE
KIN-201 Template category form

### PR DESCRIPTION
**Pull request by FernandoJoachin**
[KIN-201](https://loopcrack.atlassian.net/browse/KIN-201) Small [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=loopcrack-org&utm_campaign=badge_info)

The template category form was created

### Problem

As developers, we want to not repeat the code and be able to reuse the same form both to create and edit categories.

### Solution

Create a template for the create and edit category form.

### Requirements checklist

The form must get the next information:

- [x] Category name
- [x] Related category icon
- [x] Category image, which will be useful for the header of the product catalog page. The input must be a drag a drop
- [x] Category tags

### Testing

- [x] The create category form must be correct.
- [x] In the viewCategoryCreate method of CtrlCategory send an array with the information that you want the inputs to display and this must be able to be displayed in the view. For example: [ "category" => ["title" => "Telemetry", "tags" => "tag1,tag2"] ]

[KIN-201]: https://loopcrack.atlassian.net/browse/KIN-201?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by CheloChidoone — APPROVED** [review]
Looks good, well done!

File: app/Views/templates/admin/categoriesFormTemplate.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    <div class="col-md-6">
+        <div class="card">
+            <div class="card-header">
+                <label class="form-label mb-0" for="category-title-input">
+                    <h5 class="card-title mb-0">Titulo de la categoría</h5>
+                </label>
+            </div>
+            <div class="card-body">
+                <div class="hstack gap-3 align-items-start">
+                    <input type="text" class="form-control" name="category_title" id="category-title-input" value="<?= $category["name"] ?? "";?>" placeholder="Ingrese el título" required>
```
Comment (Cristianpan) [inline app/Views/templates/admin/categoriesFormTemplate.php:11]: Please change the values of the "name" attributes of the inputs to correspond to the name of the table columns in the database.

**Review by Cristianpan — CHANGES_REQUESTED** [review]
Everything looks good, but I think it would be better to move the form template to the folder where are the views corresponding to the category to have a better order and please make the other changes

**Review by Cristianpan — APPROVED** [review]

**Review by CarlosJCD — APPROVED** [review]

**Review by Oswaldo-Chan — APPROVED** [review]

**Review by LUGAMAFE — APPROVED** [review]
Se ve muy bien @FernandoJoachin solo hay que asegurarnos de no recibir tambien videos. Actualmente se pueden subir

**Pull request merged**